drivers/storage/portworx/component/csi.go

```diff
@@ -332,6 +326,18 @@ func (c *csi) createClusterRole(
 		},
 	}
 
+	if !pxutil.IsPrivileged(cluster) {
+		clusterRole.Rules = append(
+			clusterRole.Rules,
+			rbacv1.PolicyRule{
+				APIGroups:     []string{"security.openshift.io"},
+				Resources:     []string{"securitycontextconstraints"},
+				ResourceNames: []string{PxRestrictedSCCName},
+				Verbs:         []string{"use"},
+			},
+		)
+	}
```

Comment:
This does not look right -- shouldn't it be something like:

```go
        sccName := PxSCCName
	if !pxutil.IsPrivileged(cluster) {
            sccName := PxRestrictedSCCName
        }
	clusterRole.Rules = append(
		clusterRole.Rules,
		rbacv1.PolicyRule{
			APIGroups:     []string{"security.openshift.io"},
			Resources:     []string{"securitycontextconstraints"},
			ResourceNames: []string{sccName},
			Verbs:         []string{"use"},
		},
	)
```

Reply:
done